README.md

```diff
@@ -181,6 +181,23 @@ Class names like `Options` and `Level` might be too common, if you get a conflic
     })
     ...
 
+#### Angular Universal
+
+To use it with Angular Universal add angular2-logger to App.module normaly and in node.module provide these custom options.   
+
+```javascript
+import { Options } from "angular2-logger/core";
+
+@NgModule({
+  bootstrap: [ AppComponent ],
+  imports: [ AppModule, ... ],
+  providers: [
+    { provide: Options, useValue: { global: false, store: false } }, // stop angular2-logger from defining global variables.
```

Comment:
There will be two configurations for ng2-logger, one for the client and one for the server. In the server the user can not set store = true because there is no localStorage available. In the client the user can either set it to true or false and everything will continue working as before.